spec/adapters/googlecharts_spec.rb

```diff
@@ -30,5 +59,62 @@
     it "Table class must be GoogleVisualr::DataTable " do
       expect(Daru::View::Table.new.table).to be_a GoogleVisualr::DataTable
     end
+    it "Table class must be GoogleVisualr::DataTable when data objects are" \
+       " of class Daru::Vector" do
+      expect(Daru::View::Table.new(
+        @data_vec1,
+        @options).table
+      ).to be_a GoogleVisualr::DataTable
+      expect(Daru::View::Table.new(
+        @data_vec1,
+        @options).options
+      ).to eq @options
+      expect(Daru::View::Table.new(
+        @data_vec1,
+        @options).data
+      ).to eq @data_vec1
+    end
+    it "Table class must be GoogleVisualr::DataTable when data objects are" \
+       " of class Daru::DataFrame" do
+      expect(Daru::View::Table.new(
+        @data_df,
+        @options).table
+      ).to be_a GoogleVisualr::DataTable
+      expect(Daru::View::Table.new(@data_df, @options).options).to eq @options
+      expect(Daru::View::Table.new(@data_df, @options).data).to eq @data_df
+    end
+    it "Table class must be GoogleVisualr::DataTable when data objects are" \
+       " of class Array" do
+      expect(Daru::View::Table.new(
```

Comment:
Isn't better to put `Daru::View::Table.new(@data_array2,@ options)` into one variable and use it in next 2 line rather than executing it again and again?  Follow it in all cases.